tutorials/server/maas-esxi-quickstart/maas-esxi-quickstart.md

```diff
@@ -0,0 +1,167 @@
+---
+id: maas-esxi-quickstart
+summary: Learn how to deploy ESXi with MAAS. MAAS is an enterprise-grade infrastructure management tool.
+categories: server
+tags: maas, esxi, virtualization, cloud, open source, infrastructure
+difficulty: 3
+status: published
+published: 2019-08-09
+author: Andres Rodriguez <andreserl@ubuntu.com>
+feedback_url: https://github.com/canonical-websites/tutorials.ubuntu.com/issues
+
+---
+
+# Provision ESXi with MAAS
+
+## Overview
+Duration: 0:05
+
+In this tutorial, you will learn how to create MAAS images of ESXi using MAAS provided packer scripts.
+
+ESXi support was first introduced in MAAS 2.5, but expanded in 2.6 allowing for the configuration of VMFS datastores, registration to vCenter and others. For more information about what features MAAS support, please refer to the following blog post <Provisioning ESXi with MAAS: An Overview> ..
+
+In order to create ESXi images MAAS leverages the use of packer, and alongside the image creation process, it introduces a series of scripts and dependencies that makes it possible for MAAS to customize each ESXi deployment, including network configuration, datastore creation and user level customization.
+
+### What you'll learn
```

Comment:
```suggestion
### You will learn how to:
```

Reply:
"What you'll learn" generates a custom styling (:heavy_check_mark:) for the bullet list. Let's keep it consistent with "most" other tutorials.